documents/read_acls.go

```diff
@@ -388,8 +388,15 @@ func (cd *CoreDocument) ATGranteeCanRead(ctx context.Context, idService identity
 		return ErrReqDocNotMatch
 	}
 	// validate that the public key of the granter is the public key that has been used to sign the access token
-	// TODO provide the time for validation here using the signature timestamp
-	err = idService.ValidateKey(ctx, granterID, at.Key, &(identity.KeyPurposeSigning.Value), nil)
+	doc, err := docService.GetVersion(ctx, cd.Document.DocumentIdentifier, at.DocumentVersion)
```

Comment:
lets not send the docSrv here. Instead, send the model to the function. This makes it easier for testing and separating the concern